### PR TITLE
Upgrade to Riak 2.0

### PR DESCRIPTION
**Pull request by kuenishi**
CC: https://github.com/basho/riak_cs/pull/992

### CODE REVIEW

File: riak_test/bin/rtdev-build-releases.sh [hunk 1]
```diff
@@ -16,10 +16,9 @@
 # Or, alternatively, just substitute the paths to the kerl install paths as
 # that should work too.
 
-# R14B04=${R14B04:-$HOME/erlang-R14B04}
-# R15B01=${R15B01:-$HOME/erlang-R15B01}
-R14B04=${R14B04:-$HOME/erlang/R14B04-64}
 R15B01=${R15B01:-$HOME/erlang/R15B01-64}
+R16B02=${R16B02:-$HOME/erlang/R16B02-64}
+: ${RTCS_DEST_DIR:="$HOME/rt/stanchion"}
```
Comment (shino) [inline riak_test/bin/rtdev-build-releases.sh]: Should this variable be `RTSTANCHION_DEST_DIR` for the consistency with `rtdev-current.sh`?

Comment (shino) [inline riak_test/bin/rtdev-build-releases.sh]: `setup()` is not used? If so, we can remove this variable.

**Comment by kuenishi** [comment]
Fixed.

**Comment by shino** [comment]
:+1: 

Great step!

**Pull request merged**